src/EnergyPlus/WeatherManager.cc

```diff
@@ -9455,14 +9455,13 @@ namespace WeatherManager {
         Array1D<Real64> MonthlyAverageDryBulbTemp(12, 0.0); // monthly-daily average outside air temperature
 
         if (!this->OADryBulbWeatherDataProcessed) {
-            const auto statFileExists = FileSystem::fileExists(state.files.inputWeatherFilePath.filePath);
+            const auto statFileExists = FileSystem::fileExists(state.files.inStatFilePath.filePath);
```

Comment:
That's the original issue (which is also my fault, https://github.com/NREL/EnergyPlus/commit/360dc3c8c1fe26dbd32128042dab68ebdcdb5a66#diff-bb21b832ce448bc4fac4bc1f849830d9094fabc89731296bd5b3b0e67097782bR10519)